inference/server/oasst_inference_server/routes/chats.py

```diff
@@ -270,3 +271,12 @@ async def handle_update_title(
     except Exception:
         logger.exception("Error when updating chat title")
         return fastapi.Response(status_code=500)
+
+
+@router.put("/{chat_id}/hide")
+async def hide_chat(
+    chat_id: str,
+    ucr: UserChatRepository = Depends(deps.create_user_chat_repository),
+):
+    await ucr.hide_chat(chat_id)
```

Comment:
can we make this a toggle? i.e. hide / unhide, either as a boolean parameter (default hidden) or simply inverting whatever is the database.

Motivation: users would expect the ability to make stuff visible again, unlike delete, which is gone permanently.